server/jvm/build.gradle.kts

```diff
@@ -13,24 +13,19 @@ subprojects  {
 
     dependencies {
         implementation(platform("global.genesis:genesis-bom:${properties["genesisVersion"]}"))
-        implementation("org.agrona:agrona:1.10.0!!")
-        testImplementation("org.jetbrains.kotlin:kotlin-test-junit:1.7.10")
+        testImplementation("org.jetbrains.kotlin:kotlin-test-junit:1.9.0")
```

Comment:
should we just rely on the version provided by the kotlin plugin? 1.9.0 is different than 1.9.10. You should also be able to use the special kotlin dependency syntax:
`testImplementation(kotlin("test-junit"))`